src/plugins/telemetryTable/TelemetryTableRow.js

```diff
@@ -88,6 +89,15 @@ define([], function () {
     getContextMenuActions() {
       return ['viewDatumAction', 'viewHistoricalData'];
     }
+
+    updateWithDatum(updatesToDatum) {
+      const normalizedUpdatesToDatum = createNormalizedDatum(updatesToDatum, this.columns);
+      this.datum = {
+        ...this.datum,
+        ...normalizedUpdatesToDatum
+      };
+      this.fullDatum = this.datum;
```

Comment:
This needs to happen before the normalization step, otherwise `this.fullDatum` just becomes a duplicate of `this.datum`

Reply:
fullDatum is before normalizing. Got it!